browser/ui/brave_shields_data_controller.h

```diff
@@ -10,10 +10,13 @@
 #include <string>
 
 #include "base/observer_list.h"
+#include "brave/components/brave_shields/common/brave_shields_panel.mojom-shared.h"
```

Comment:
you shouldn't have to include the `-shared` header, were you getting an error here?

Reply:
yea, this isn't needed